test/Tests/polyglot-sources/enso-test-java-helpers/src/main/java/org/enso/base_test_helpers/IntHolder.java

```diff
@@ -0,0 +1,9 @@
+package org.enso.base_test_helpers;
+
+public final class IntHolder {
+  public final Integer value;
```

Comment:
Out of interest why box from `int` to `Integer`?

Reply:
Good question. Let's test both variants: a311e6345